dotCMS/src/main/java/com/dotcms/contenttype/transform/contenttype/DbContentTypeTransformer.java

```diff
@@ -133,6 +133,18 @@ public BaseContentType baseType() {
 				return base;
 			}
 
+			@Override
+			public String icon() {
+				final String icon = (String) map.get("icon");
```

Comment:
since it is a map you can do
return (String)map.getOrDefault("icon", null)

Reply:
Well, it would depending: if you want to return null even when the value is an empty string, your logic is ok